.github/workflows/build.yml

```diff
@@ -214,28 +214,8 @@ jobs:
         uses: actions/checkout@v3
       - name: Setup
         uses: ./.github/actions/setup
-      - name: Run all tests other than core
-        run: npx nx run-many --target=test --all --exclude=core
-      - name: Make test report dir
-        run: mkdir -p reports/junit/
-        working-directory: packages/core/
-      - name: Run core tests and generate test data
-        run: npx nx run core:coverage
-      - name: Move test report
-        run: mv junit.xml reports/junit/
-        working-directory: packages/core/
-      - name: Upload coverage report to codecov
-        uses: codecov/codecov-action@v3
```

Comment:
Given that we're deleting these lines, should we also delete `codecov.yml` from the repo root?

Reply:
yeah, I don't think historical coverage data help us much so it's not worth the maintenance

Reply:
done